### PR TITLE
feat(content): Pneumatic Fun Reintegration

### PR DESCRIPTION
**Pull request by yay855**
For years pneumatic weapons have been locked behind a mod. No more!

## Purpose of change

Pneumatic weapons served as a useful earlygame ranged option, but were obsoleted way back when. The Pneumatic Fun mod reimplemented them, but these weapons are useful earlygame ranged options.

## Describe the solution

The contents of the Pneumatic Fun mod, barring the meme-y pebble ammos, have been reimplimented, with some minor adjustments to damage, range, and ammo recovery for the nail weapons and ammos.

### CODE REVIEW

**Comment by RobbieNeko** [comment]
Two of the rail rifles are specifying the same sound value as they're copying from, one of the nail guns is specifying the same skill as its copyfrom, and the nailmag is having some general issues with not being a magazine and apparently circular dependencies.

File: data/json/items/gun/metal_rail.json [hunk 1]
```diff
@@ -1,50 +1,69 @@
 [
   {
     "id": "heavy_rail_rifle",
-    "copy-from": "rifle_magnetic",
     "type": "GUN",
+    "copy-from": "rifle_magnetic",
+    "reload_noise_volume": 10,
```
Comment (RobbieNeko) [inline data/json/items/gun/metal_rail.json:6]:
```suggestion
    
```

File: data/json/items/gun/metal_rail.json [hunk 2]
```diff
@@ -1,50 +1,69 @@
...
     "ammo_effects": [ "INCENDIARY", "TRAIL" ],
-    "extend": { "flags": [ "RELOAD_ONE", "MOUNTED_GUN" ] }
+    "flags": [ "RELOAD_ONE" ]
   },
   {
     "id": "rebar_rifle",
-    "copy-from": "rifle_magnetic",
     "type": "GUN",
+    "copy-from": "rifle_magnetic",
+    "reload_noise_volume": 10,
```
Comment (RobbieNeko) [inline data/json/items/gun/metal_rail.json:40]:
```suggestion
    
```

File: data/json/items/gun/nail.json [hunk 3]
```diff
@@ -14,10 +14,73 @@
...
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": "3341 g",
+    "volume": "2750 ml",
+    "price": "750 USD",
+    "price_postapoc": "50 USD",
+    "to_hit": -2,
+    "bashing": 10,
+    "material": [ "copper", "steel" ],
+    "ammo": "nail",
+    "skill": "rifle",
```
Comment (RobbieNeko) [inline data/json/items/gun/nail.json:41]:
```suggestion
    
```

File: data/json/items/magazine/nail.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "id": "nailmag",
+    "looks_like": "stanag30",
+    "copy-from": "nailmag",
```
Comment (RobbieNeko) [inline data/json/items/magazine/nail.json:5]: why in the world is it copy-from itself o.O

**Review by RobbieNeko — COMMENTED** [review]
Here's pointing out the issues. Looks like some silly jank when it was brought into a mod that just never got caught, mostly.

File: data/json/items/gun/metal_rail.json (hunk 1 above)
Comment (RobbieNeko) [reply data/json/items/gun/metal_rail.json]: Note: intention is not to leave blank lines, I just don't know exactly how much of the suggestion formatting I'd need to actually just straight up suggest removal haha

File: data/json/items/magazine/nail.json (hunk 4 above)
Comment (yay855) [reply data/json/items/magazine/nail.json]: I have no clue, but it's resolved now.

**Comment by RobbieNeko** [comment]
There we go, that's much better. Now the tests shouldn't yell at you ^-^

**Review by chaosvolt — COMMENTED** [review]
Is it supposed to basically undo the changes made in https://github.com/cataclysmbnteam/Cataclysm-BN/pull/2718?

**Comment by VissValdyr** [comment]
On another note, nice to see another very active jsoner! Good job @yay855

**Comment by yay855** [comment]
> Is it supposed to basically undo the changes made in #2718?

Whoops, my bad! I'll undo that undo.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**